### PR TITLE
MGMT-2634 Remove host-name flag

### PR DESCRIPTION
**Pull request by carbonin**
This is not used as of 93495e81245278863102d155b15b4d0782641ff4 and is no longer being sent from the service as of openshift/assisted-service#670

### CODE REVIEW

**Comment by carbonin** [comment]
/hold

until openshift/assisted-service#670 is merged

**Review by YuviGold — APPROVED** [review]

**Pull request merged**
